src/components/edit-listing-schedule/DateInput/index.jsx

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { DateInput } from '../ListingSchedule/edit-listing-schedule';
+
+/**
+ *
+ * @param  {function} { setDateRangeValue } to get the start end date range value
+ *
+ * @return input component for the start end date range value
+ */
+function DatePickerInput({ setDateRangeValue }) {
+  const [dateRange, setDateRange] = useState([null, null]);
+  const [startDate, endDate] = dateRange;
+  return (
+    <DateInput
+      placeholderText="10-05-2021  - 16-05-2021"
```

Comment:
```suggestion
      placeholderText="Click to select a date"
```
I feel this can get confusing for users.